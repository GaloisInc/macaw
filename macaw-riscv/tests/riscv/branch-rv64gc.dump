
branch-rv64gc.exe:     file format elf64-littleriscv


Disassembly of section .text:

000000000001010c <main>:
   1010c:	1101                	c.addi	sp,-32
   1010e:	ec22                	c.sdsp	s0,24(sp)
   10110:	1000                	c.addi4spn	s0,sp,32
   10112:	fec42783          	lw	a5,-20(s0)
   10116:	2781                	c.addiw	a5,0
   10118:	e781                	c.bnez	a5,10120 <main+0x14>
   1011a:	4785                	c.li	a5,1
   1011c:	fef42623          	sw	a5,-20(s0)
   10120:	fec42783          	lw	a5,-20(s0)
   10124:	853e                	c.mv	a0,a5
   10126:	6462                	c.ldsp	s0,24(sp)
   10128:	6105                	c.addi16sp	sp,32
   1012a:	8082                	c.jr	ra
