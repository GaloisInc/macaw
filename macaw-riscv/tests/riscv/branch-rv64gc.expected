R { fileEntryPoint = Nothing
  , funcs = [(0x1010c, [(0x1010c,14), (0x1011a,6), (0x10120, 12)])]
  , ignoreBlocks = []
  }
